dll/cpl/appwiz/lang/ro-RO.rc

```diff
@@ -29,32 +29,32 @@ END
 
 IDD_SHORTCUT_LOCATION DIALOGEX 0, 0, 340, 200
 STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
-CAPTION "Expertul pentru creare de scurtături"
+CAPTION "Creare comandă rapidă"
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
-    LTEXT "Acest expert vă va ajuta în crearea unei scurtături\ncătre adrese din Internet, calculatoare, dosare, fișiere\nlocale sau din rețea.", -1, 120, 15, 200, 30
-    LTEXT "Scrieți adresa elementului:", -1, 120, 50, 120, 10
+    LTEXT "Acest asistent vă va ajuta \în crearea unei scurtături\ncătre adrese din Internet, calculatoare, dosare, fișiere\nlocale sau din rețea.", -1, 120, 15, 200, 30
```

Comment:
what is the purpose of that added backslash? Why wasn't it necessary before?

Reply:
resolved.

Reply:
I added it by mistake.